blood_donation_app/templates/base.html

```diff
@@ -5,6 +5,7 @@
     <meta charset="utf-8">
     <meta content="width=device-width, initial-scale=1, shrink-to-fit=no" name="viewport">
     {% block meta_tags %}{% endblock %}
+    <link rel="icon" type="image/png" href="/static/blood.jpeg"/>
```

Comment:
Instead of manually typing the path of the static file, you should use `static` templatetag as per the reference.

ref: https://docs.djangoproject.com/en/3.2/howto/static-files/